src/Admin/AdminHelper.php

```diff
@@ -202,7 +202,7 @@ public function appendFormFieldElement(AdminInterface $admin, $subject, $element
 
         //Child form not found (probably nested one)
         //if childFormBuilder was not found resulted in fatal error getName() method call on non object
-        if (!$childFormBuilder) {
+        if (!$childFormBuilder || !$admin->hasFormFieldDescription($childFormBuilder->getName())) {
```

Comment:
Can you update the comment before to add better explanation ?

Reply:
:ok_hand: 

BTW PhpStorm suggested to me to flip `if-else`.
And if will look prettier to me

```php
if ($childFormBuilder && $admin->hasFormFieldDescription($childFormBuilder->getName())) {
``` 

But I think PR should atomic, maybe in next PR

Reply:
It's not a bad idea to flip indead !

Reply:
ok, I've flipped it